ctapipe/tools/muon_reconstruction.py

```diff
@@ -75,39 +67,36 @@ class MuonAnalysis(Tool):
 
     aliases = {
         ("i", "input"): "EventSource.input_url",
-        ("o", "output"): "MuonAnalysis.output",
+        ("o", "output"): "DataWriter.output_path",
         ("m", "max-events"): "EventSource.max_events",
         ("t", "allowed-tels"): "EventSource.allowed_tels",
     }
 
     flags = {
-        "f": ({"MuonAnalysis": {"overwrite": True}}, "Overwrite output file"),
+        "f": (
+            {"DataWriter": {"overwrite": True}},
+            "Overwrite output file if it exists",
+        ),
         **flag(
             "overwrite",
-            "MuonAnalysis.overwrite",
-            "Overwrite output file",
-            "Don't overwrite output file",
+            "DataWriter.overwrite",
+            "Overwrite output file if it exists",
+            "Don't overwrite output file if it exists",
         ),
     }
 
     def setup(self):
-        if self.output is None:
-            raise ToolConfigurationError("You need to provide an --output file")
-
-        if self.output.exists() and not self.overwrite:
-            raise ToolConfigurationError(
-                "Outputfile {self.output} already exists, use `--overwrite` to overwrite"
-            )
-
         self.source = EventSource(parent=self)
         subarray = self.source.subarray
 
         self.calib = CameraCalibrator(subarray=subarray, parent=self)
         self.ring_fitter = MuonRingFitter(parent=self)
         self.intensity_fitter = MuonIntensityFitter(subarray=subarray, parent=self)
         self.cleaning = TailcutsImageCleaner(parent=self, subarray=subarray)
```

Comment:
Should the cleaner be configurable?

Reply:
After this, the moon tool should be removed, as running process without storing images and parameters will be the same

Reply:
Or rather as part of this pr